test/paratii.eth.js

```diff
@@ -109,6 +113,14 @@ describe('paratii.eth API: :', function () {
     assert.equal(balance1 - balance0, amount)
   })
 
+  it('subscription to Tranfer PTI events should work as expected', async function () {
```

Comment:
CAn we add here a "TODO" that this tests is not finished (we should at least check that the subscription indeed triggers the given callback.